packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -103,6 +103,10 @@ const PopoverWrapper = forwardRef((props: PopoverWrapperProps, ref: RefObject<HT
     isDisabled: isNonModal
   });
 
+  if (arrowProps?.style.left === undefined && arrowProps?.style.top === undefined) {
+    hideArrow = true;
+  }
```

Comment:
Rather than setting `hideArrow = true`, would it make sense to use  `arrowProps.style.visibility = 'hidden';`? This way, any change in `arrowProps?.style.left` and `arrowProps?.style.top` based on the overlay position changing will not result in an abrupt shift in the offset position.

Reply:
I thought I talked in Slack about exploring a solution like this and rejected it because `visibility` is higher in the call stack and the logic to set it would be like this or would require a new value in the return value a might change the API.